packages/react/src/_collapsing-side-bar/CollapsingSideBar.tsx

```diff
@@ -69,136 +69,122 @@ export function CollapsingSideBar({
 		},
 	});
 
-	const { windowWidth } = useWindowSize();
-	const isMobile = (windowWidth || 0) <= tokens.breakpoint.lg - 1;
-
 	return (
-		<CollapsingSideBarContainer
+		<Stack
 			as={as}
+			aria-labelledby={headingId}
 			background={background}
-			ariaLabel={ariaLabel}
+			css={{
+				[collapsingSideBarHoverProp]: hoverColorMap[background],
+			}}
+			gap={gap}
 		>
-			{title}
-			<SideBarCollapseButton
-				isOpen={isOpen}
-				onClick={onToggle}
-				ariaControls={bodyId}
-				id={buttonId}
-			>
-				{collapseButtonLabel}
-			</SideBarCollapseButton>
+			<Box css={{ position: 'relative' }}>
+				<Stack
+					css={{
+						// Make the accesible heading appear the same size as the button for a screen reader's virtual cursor
+						inset: 0,
+						position: 'absolute',
+						[tokens.mediaQuery.min.md]: {
+							position: 'static',
+						},
+					}}
+					gap={0.5}
+					paddingX={customTitle ? undefined : { xs: 0.75, md: 0 }}
+					paddingY={customTitle ? undefined : { xs: 1, md: 0 }}
+					id={headingId}
+				>
+					{customTitle || (
+						<>
+							<Text
+								as="h2"
+								color="text"
+								fontSize="md"
+								fontWeight="bold"
+								lineHeight="heading"
+							>
+								{title}
+							</Text>
+							{subTitle && (
+								<Text color="muted" fontSize="xs">
+									{subTitle}
+								</Text>
+							)}
+						</>
+					)}
+				</Stack>
+				<Box
+					aria-controls={bodyId}
+					aria-describedby={headingId}
+					aria-expanded={isOpen}
+					aria-label={`Open ${title}`}
```

Comment:
I feel the `Open` is not needed here. We already get the announcement of expanded/collapsed button, it adds hidden text to the start of the visible title and it always says "open" even when the thing is already open - in which case pressing it would close the sidebar.

Reply:
My thinking was that it was the same as the Main nav Menu button and also to separate it from the heading, so users aren't left wondering why there's a heading followed by a button with the same accessible name.

Reply:
>same as the Main nav Menu button

The difference here is that the main menu button has a single function, so "Open" makes sense. Close is a separate button within the dialog.

>also to separate it from the heading

Yeah, that's a good point. What about:

```ts
aria-label={`${isOpen ? 'Close' : 'Open'} ${title}`}
```

Or if changing the accessible name for a button in this way, at this time is bad, maybe:

```ts
aria-label={`${title}, toggle open/close.`}
```

I'm just not convinced leaving this as "Open..." is the best UX, though I get the reasoning.

Reply:
> I'm just not convinced leaving this as "Open..." is the best UX, though I get the reasoning.

Same. Especially for voice dictation who may wonder why they can't speak "Progress indicator" and find the button.

Just remembered that I changed the main nav's Menu button for the same reason to "Menu (Open main menu)", so perhaps something like `${title} (Expand/Collapse)`?

Reply:
Yep, either works for me!